ObjCVoiceCallKitQuickstart/ViewController.m

```diff
@@ -308,13 +316,14 @@ - (void)provider:(CXProvider *)provider performAnswerCallAction:(CXAnswerCallAct
     //      `provider:performAnswerCallAction:` per the WWDC examples.
     // [[TwilioVoice sharedInstance] configureAudioSession];
 
-    self.call = [self.callInvite acceptWithDelegate:self];
-    if (self.call) {
-        self.call.uuid = [action callUUID];
-    }
+    [self performAnswerVoiceCallWithUUID:action.callUUID completion:^(BOOL success) {
+        if (success) {
+            [action fulfill];
+        } else {
+            [action fail];
+        }
+    }];
     
-    self.callInvite = nil;
-
     [action fulfill];
```

Comment:
@bchen-twilio I was just wondering whether this last `[action fulfill]` is correct or it is left from the refactor. Because I see that is is called inside the success completion block.

Reply:
Hi @fabriziomoscon 

For some reason, I had to keep `[action fulfill]` in the `provider:performAnswerCallAction:` method otherwise the Voice iOS SDK won't be able to initialize & start the call audio properly.

-bobie